react/features/base/logging/JitsiMeetLogStorage.ts

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable lines-around-comment */
+// @ts-ignore
+import RTCStats from '../../rtcstats/RTCStats';
```

Comment:
Is this import actually still needed? I don't see it being used anywhere.

Reply:
Yes, around line 100 where it's used to invoke the new send logs method `RTCStats.sendLogs(logEntries);`